src/lib/store.js

```diff
@@ -1,98 +1,54 @@
-import { writePublisherConsentCookie, writeVendorConsentCookie, encodePublisherConsentData, encodeVendorConsentData } from "./cookie/cookie";
+import {
+	writeConsentCookie,
+	decodeConsentData,
+	encodeConsentData
+} from "./cookie/cookie";
 import config from './config';
 import { findLocale } from './localize';
 import log from './log';
-const arrayFrom = require('core-js/library/fn/array/from');
+import { GVL, TCModel, Vector } from '@iabtcf/core';
 
 export const SECTION_INTRO = 0;
 export const SECTION_DETAILS = 1;
 export const SECTION_PURPOSES = 0;
 export const SECTION_VENDORS = 1;
 
-/**
- * Copy a data object and make sure to replace references
- * of Set objects with new ones.
- */
-function copyData(dataObject) {
-	if (typeof dataObject !== 'object') {
-		return dataObject;
-	}
-	const copy = {...dataObject};
-	for (let key in copy) {
-		if (copy.hasOwnProperty(key) && copy[key] instanceof Set) {
-			copy[key] = new Set(copy[key]);
-		}
-	}
-	return copy;
-}
-
 export default class Store {
 	constructor({
-		cmpId = 1,
-		cmpVersion = 1,
-		cookieVersion = 1,
-		vendorConsentData,
-		globalVendorConsentData,
-		publisherConsentData,
-		vendorList,
-		customPurposeList,
-		pubVendorsList,
-		allowedVendorIds
+		cmpId = 280,
```

Comment:
Maybe it would be good idea to remove default value from here or leave `1` (value `1` is considered as test ID).

Reply:
It is impossible to create TCModel instance with cmpId = 1

Reply:
OK, so we may change this parameter as required without any default value

Reply:
done